class EmailAlreadyInUse(Exception):
    pass

class UserNotFound(Exception):
    pass